tests.py

```diff
@@ -169,3 +169,42 @@ class Pkg(ConanFile):
       run("cmake .. -DCMAKE_BUILD_TYPE=Release")
       run("cmake --build . --config Release")
       run("bin\main")
+
+    def test_exported_package(self):
+      content = """cmake_minimum_required(VERSION 2.8)
+project(conan_wrapper CXX)
+
+set(CONAN_EXPORTED ON)
```

Comment:
I guess we should remember to remove this ``CONAN__EXPORTED ON`` line, when the conan codebase already sets it automatically.